Packs/MicrosoftExchangeOnline/ReleaseNotes/1_3_4.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### EWS O365
+
+- Updated the Docker image to: *demisto/py3ews:1.0.0.91864*.
+- Improved implementation of the code.
```

Comment:
better to give a less vague description

Reply:
I'm unsure how to document their changes made to the code; it's simply reverting with no impact on clients. I used something I found in our repository. Any suggestions?